trunk/SUAVE/Methods/Weights/Buildups/Common/fuselage.py

```diff
@@ -0,0 +1,169 @@
+## @ingroup Methods-Weights-Buildups-Common
+
+# fuselage.py
+#
+# Created: Jun, 2017, J. Smart
+# Modified: Apr, 2018, J. Smart
+
+#-------------------------------------------------------------------------------
+# Imports
+#-------------------------------------------------------------------------------
+
+from SUAVE.Core import Units
+from SUAVE.Attributes.Solids import (
+    Bidirectional_Carbon_Fiber, Honeycomb, Paint, Unidirectional_Carbon_Fiber, Acrylic, Steel)
+import numpy as np
+
+
+#-------------------------------------------------------------------------------
+# Fuselage
+#-------------------------------------------------------------------------------
+
+## @ingroup Methods-Weights-Buildups-Common
+def fuselage(config,
+             maximum_g_load = 3.8,
+             landing_impact_factor = 3.5,
+             safety_factor = 1.5):
+    """ weight = SUAVE.Methods.Weights.Buildups.Common.fuselage(
```

Comment:
Is this part of comment necessary?

Reply:
I believe so. The Documentation guide only says that this section should be a description of the function, and I think a full location of the function and the default values are good, immediate references that someone would want to be able to quickly check on the documentation.